forem/forem
# Update the Sidebar to include the Member Manager

## What type of PR is this? (check all applicable)


## Description
This creates a `member_manager` navigation item that contains `members`, `invitations` and `gdpr_delete_requests`. It also moves the badge to the `gdpr_delete_requests`.

**BEFORE**: 
<img width="1512" alt="Screenshot 2022-05-04 at 09 49 39" src="https://user-images.githubusercontent.com/2786819/166641664-98d6e75e-d4cb-43d1-b284-cd20fb7046fc.png">

**AFTER**:
<img width="1512" alt="Screenshot 2022-05-05 at 11 11 52" src="https://user-images.githubusercontent.com/2786819/166894298-52ad3974-ac86-47fc-8268-1c50cb5f7fca.png">

**Notes**:
When the GDPR badge is over 9, it moves it to the next line. I'm not sure if this is the ideal situation and still querying it [here](https://forem-team.slack.com/archives/C01S4QZUDNC/p1651650256078729). 
<img width="1511" alt="Screenshot 2022-05-02 at 14 04 00" src="https://user-images.githubusercontent.com/2786819/166307750-2a3fac77-8a23-40fa-9cd1-38606039e057.png">

## Related Tickets & Documents

- Related Issue #
- Closes #https://github.com/forem/forem/issues/17460
- Closes #https://github.com/forem/forem/issues/17461
- Closes #https://github.com/forem/forem/issues/17462
- Closes #https://github.com/forem/forem/issues/17463
- Closes #https://github.com/forem/forem/issues/17464

## QA Instructions, Screenshots, Recordings

Screenshots provided in the description. 

1. Please click around on the member manager sidebar item and ensure that the correct tabs get highlighted.
2. Delete a user by clicking on "Delete Users" in the three dot menu on `/admin/users/<user-id>`. You should see the badge increase on GDPR Delete Requests. Ensure 

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/hqPUQW4z3VIeFEOsFa/giphy.gif)
